recipes/rainbow/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: rainbow
+  version: "2.0.4"
+source:
+  fn: rainbow_2.0.4.tar.gz
+  url: https://sourceforge.net/projects/bio-rainbow/files/rainbow_2.0.4.tar.gz/download
```

Comment:
I guess this one is a little bit more sustainable: http://downloads.sourceforge.net/project/bio-rainbow/rainbow_2.0.4.tar.gz